pyproject.toml

```diff
@@ -19,9 +18,9 @@ classifiers = [
 ]
 keywords = "mdformat,markdown,formatter,gfm"
 
-requires-python=">=3.7"
+requires-python=">=3.9"
 requires=[
-    "mdformat >=0.7.0,<0.8.0",
+    "mdformat >=0.7.0",
```

Comment:
We shouldn't need to pin `mdformat` because a breaking change may not impact this package